packages/react-instantsearch-hooks-web/src/ui/__tests__/ShowMoreButton.test.tsx

```diff
@@ -4,9 +4,19 @@ import React from 'react';
 
 import { ShowMoreButton } from '../ShowMoreButton';
 
+const defaultTranslations = {
+  showMore: 'Show more',
+  showLess: 'Show less',
```

Comment:
Not a fan, because if you want a single text (e.g., "Toggle") you'll have to put it twice.

Reply:
But the default for us are 2 strings, I guess users wanting one string for both is a marginal use case while most users just want 2 different strings ?

Reply:
I don't know about that assumption, we don't really have a good way to know.

In any case, if we want two string translations they should be called `showMoreButtonText` and `showLessButtonText` to fit the [naming scheme](https://github.com/algolia/instantsearch-rfcs/blob/master/accepted/translations-api-rish.md#naming) we chose. But then again, there's kind of a mismatch between that and our CSS classes because there's no "show less" button.

I still believe the function is the best pick because the code to write is really straightforward, and the translation key matches a single element, like the others.

If users do internationalization with this API, they will have a lot of logic as well, so I'm not worried that having to write one ternary will be considered a lot of work.

Reply:
One last thing I'm wondering is, let's say the developer wants to just display "Toggle", there would be no way for users to know what state the button is in, we would either have to also add a CSS class or an icon, or two strings like "Toggle (⬆)" and "Toggle (⬇️)"

Also in the case of a function should the argument be an object with a single `isShowingMore`  boolean or just directly this boolean ? It's an object in the RFC

Reply:
Makes sense, I'll do the same for the functions in `<Pagination>` then :)